hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -312,6 +344,18 @@ def _make_test(imports: Set[str], body: str) -> str:
     return black.format_str(header + body, mode=black.FileMode())
 
 
+def _is_probably_ufunc(obj):
+    # See https://numpy.org/doc/stable/reference/ufuncs.html - there doesn't seem
+    # to be an upstream function to detect this, so we just guess.
+    has_attributes = "nin nout nargs ntypes types identity signature".split()
+    not_attributes = ("__qualname__", "__module__")
```

Comment:
Is there any reason why we would expect that ufuncs won't have a qualname in the future? More generally, why are these useful attributes to avoid given the other checks that we are doing?

Reply:
Fair enough - I'll take out the not-clause.